### PR TITLE
Widget/rework of card design

### PR DESCRIPTION
**Pull request by jdsmith022**
Redesign of hover card 

current size: 350 X 435
added new pngs designed specifically for douchezaak.nl
final copy still needs to be confirmed and added

minor adjustment to placeWidgetInfoBox to ensure the card is not clipped on different res

current design:

Screen Shot 2020-09-22 at 3 11 45 PM
Screen Shot 2020-09-22 at 3 11 30 PM

### CODE REVIEW

File: frontend/public/widget/co2okWidgetMark-519dc2a3.js [hunk 1]
```diff
@@ -242,24 +261,28 @@ let Co2okWidgetXL = {
...
-      
+
       if (offset.left < 0) offset.left = 10;
       if (offset.top < 0) offset.top = 10;
-      if (offsetMobile.left < 0) offsetMobile.left = 10;
+      if (offsetMobile.left < 0) offsetMobile.left = 5;
       if (offsetMobile.top < 0) offsetMobile.top = 10;
 
-      if (jQuery(window).width() < 480) {
+      let window_width = jQuery(window).width();
```
Comment (Mil0dV) [inline frontend/public/widget/co2okWidgetMark-519dc2a3.js:282]: As discussed, these 2 lines can be removed

**Review by Mil0dV — CHANGES_REQUESTED** [review]
Looks good to me!

**Comment by jdsmith022** [comment]
Screen Shot 2020-09-25 at 12 19 35 PM

Screen Shot 2020-09-25 at 12 19 45 PM

**Review by Ghostyfied — APPROVED** [review]
No changes needed from my end.

**Pull request closed without merging**